library/ecp_internal.h

```diff
@@ -28,6 +28,18 @@
 #include "mbedtls/bignum.h"
 #include "bignum_mod.h"
 
+/** Convert an MPI to its canonical representative.
```

Comment:
I think it might worth being more specific as there will be another conversion function that takes arbitrary input (the input won't be limited to 2N like here).

```suggestion
/** Convert the result of a quasi-reduction to its canonical representative.
```